packages/spaces/types/Spaces.d.ts

```diff
@@ -17,14 +17,15 @@ export interface SpacesProps {
 export interface SpaceType {
   error?: string;
   space: object;
-  isGhost: boolean;
   loading: boolean;
 }
 
-declare function useSpace(id?: string): SpaceType;
+declare function useSpaces(ids?: string[]): object[];
```

Comment:
`useSpaces` declaration for the `...args` would be `...ids?: string`

I would create a `SpaceType` that at least contains the intellisense for `isGhost` so we know that is a valid value. In the future, I do see us having some global typescript definitions for our different domain models so probably a good place to start.

```ts
interface Space {
   [key:string]:any;
   isGhost?: boolean
}

Reply:
this is what the ts linter recommended 🤷‍♂ c0e5db88321ca406ffe197e64414c0ae9020bdea

Reply:
went ahead and created `Space` type: 86cc811cac3f2b41f064aae3bc8fadab4d696ea6